.gitignore

```diff
@@ -3,7 +3,7 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
The `.venv/` entry is correctly included to ignore virtual environment directories. However, ensure that this is the correct directory name for your virtual environment, as it might differ (e.g., `venv/`, `env/`).